### PR TITLE
Features/implement new status endpoint

### PR DESCRIPTION
**Pull request by Redm4x**
- Support for the new provider's status endpoint (grpc)
  - Keep track of nodes and cpu/gpu info (vendor/model)
  - Fallback to old tracking if provider isn't updated yet (based on the `/version` endpoint)
- Improved performance for provider graphs

### CODE REVIEW

File: indexer/src/providers/statusEndpointHandlers/grpc.ts [hunk 1]
```diff
@@ -0,0 +1,332 @@
...
+    } catch (err) {
+      reject(err);
+    }
+  });
+}
+
+const protosetBuffer = fs.readFileSync("./src/proto/akash/providerServiceDescriptor.bin");
+const descriptorSet = loadFileDescriptorSetFromBuffer(protosetBuffer);
+const packageDef = grpc.loadPackageDefinition(descriptorSet);
+const clientInsecureCreds = grpc.credentials.createInsecure();
```
Comment (baktun14) [inline indexer/src/providers/statusEndpointHandlers/grpc.ts:14]: Move those constants at the top of the file?

File: indexer/src/providers/statusEndpointHandlers/grpc.ts [hunk 2]
```diff
@@ -0,0 +1,332 @@
...
+      {
+        cpu: 0,
+        gpu: 0,
+        memory: 0,
+        storage: 0
+      }
+    );
+  const checkDate = toUTC(new Date());
+
+  console.time("updateData");
```
Comment (baktun14) [inline indexer/src/providers/statusEndpointHandlers/grpc.ts]: Is it fast?

File: indexer/src/providers/statusEndpointHandlers/grpc.ts [hunk 3]
```diff
@@ -0,0 +1,332 @@
...
+      {
+        cpu: 0,
+        gpu: 0,
+        memory: 0,
+        storage: 0
+      }
+    );
+}
+
+type NewStatusResponseType = {
```
Comment (baktun14) [inline indexer/src/providers/statusEndpointHandlers/grpc.ts]: I don't think we have one, but we should have a folder with all the types.

**Comment by Redm4x** [comment]
Added a flag to disable the new status endpoint for now, will deploy like this to get performance improvements asap and will enable grpc later on.

File: indexer/src/providers/statusEndpointHandlers/grpc.ts (hunk 3 above)
Comment (Redm4x) [reply indexer/src/providers/statusEndpointHandlers/grpc.ts]: Moved the type to a new `types` folder, with a TODO to remove it once we have grpc client generation

**Review by baktun14 — APPROVED** [review]

**Pull request merged**